dev-qt/kdsingleapplication/kdsingleapplication-1.0.0.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake
+
+DESCRIPTION="KDSingleApplication is a helper class for single-instance policy applications"
+HOMEPAGE="https://github.com/KDAB/KDSingleApplication/"
+SRC_URI="https://github.com/KDAB/KDSingleApplication/releases/download/${PV}/${P}.tar.gz"
+KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE="qt6 static"
+
+BDEPEND=""
+COMMON_DEPEND="
+	!qt6? (
+		dev-qt/qtcore:5
+	)
+	qt6? (
+		dev-qt/qtbase
+	)
```

Comment:
- This can be done on 2 lines.
- `qtcore:5`, but not `qtbase:6`?

Reply:
qtbase is only slotted for qt6, AFAICK it will never be slotted for 5, but I can add it anyway if you want

Reply:
what about Qt7?